docs/src/index.md

```diff
@@ -1,3 +1,31 @@
 # DocumenterTools.jl
 
-Documentation for DocumenterTools.jl
+This is the Documentation for DocumenterTools.jl.
+DocumenterTools.jl is a utility package to aid in setting up
+documentation generation with Documenter.jl.
+
+## Generate basic structure
+
+The DocumenterTools.generate function aids in setting up the general
+file structure that Documenter expects when building documentation.
+
+
+!!! note
+    Make sure the package is `Pkg.develop`ed before trying this,
+    since packages that are installed in the default location should not be touched.
+
+```@docs
+DocumenterTools.generate
+```
+
+## Generating SSH keys
+
+The `Travis.genkeys` function generates SSH keys that can be used to
+push the generated documentation to `gh-pages` on GitHub.
+
+!!! warning
+    Please carefully read Documenters instructions on deployment before generating keys.
+
+```@docs
+DocumenterTools.Travis.genkeys
+```
```

Comment:
Actually, maybe in order not to duplicate these docs, perhaps there should be just a link to the "Hosting documentation" page in Documenter's manual? We should avoid duplication either way, but not sure if the tutorial should be here or in Documenter.

Reply:
Maybe this package should not have any docs then? We can just direct people to Documenters docs from DocumenterTools README.

Reply:
Yea thats what I thought. That would mean Documenters docs depend on this package, but not Documenter itself, which was the intended outcome.

Reply:
Yup. So should we nuke the whole `docs/` directory here?

Reply:
Yes, I did that.